share/platform.sdks.json

```diff
@@ -0,0 +1,14 @@
+{
+    "armv7-android": ["android", "ndk25"],
+    "arm64-android": ["android", "ndk25"],
+    "js-web": ["emsdk", "3155"],
+    "wasm-web": ["emsdk", "3155"],
+    "x86_64-win32": ["winsdk", "2022"],
+    "x86-win32": ["winsdk", "2022"],
+    "x86_64-linux": ["linux", "latest"],
+    "arm64-osx": ["osx", "latest"],
+    "x86_64-osx": ["osx", "latest"],
+    "arm64-ios": ["osx", "latest"],
+    "x86_64-ios": ["osx", "latest"]
```

Comment:
As a side note, we also try to move versions to a single place (for easier updates).
Currently, the sdk.py is the most "central" place.
I'm not saying this needs to change now, but perhaps there should be a link from sdk.py to this file.